modules/apps/account/account-service/src/main/java/com/liferay/account/internal/upgrade/v1_3_0/UpgradeAccountEntry.java

```diff
@@ -41,6 +41,12 @@ protected void doUpgrade() throws Exception {
 				AccountEntryTable.class,
 				new AlterTableAddColumn("emailAddress", "VARCHAR(254) null"));
 		}
+
+		if (!hasColumn("AccountEntry", "taxExemptionCode")) {
+			alter(
+				AccountEntryTable.class,
+				new AlterTableAddColumn("taxExemptionCode", "String null"));
```

Comment:
I think `String` should be all caps: `STRING`

Reply:
The `com.liferay.account.internal.upgrade.v1_3_0.util.AccountEntryTable` also needs to be regenerated.